pkg/cache/cache.go

```diff
@@ -473,7 +473,9 @@ func resourceLimitsByName(in []kueue.Resource) map[corev1.ResourceName][]FlavorL
 			fLimits := FlavorLimits{
 				Name:       string(f.ResourceFlavor),
 				Guaranteed: workload.ResourceValue(r.Name, f.Quota.Guaranteed),
-				Ceiling:    workload.ResourceValue(r.Name, f.Quota.Ceiling),
+			}
+			if f.Quota.Ceiling != nil {
+				fLimits.Ceiling = pointer.Int64(workload.ResourceValue(r.Name, *f.Quota.Ceiling))
```

Comment:
wouldn't it be safer if we keep flimits.Ceiling as type `int64` and just set it to MaxInt when f.Quota.Ceiling is nil?

Reply:
I think pointer is more readable. It helps with setting up unit tests too.

Reply:
ok